packages/pn-pa-webapp/src/pages/components/AppStatus/useDownloadDocument.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from "../../../redux/hooks";
+import { RootState } from "../../../redux/store";
+import { clearLegalFactDocumentData } from '../../../redux/appStatus/reducers';
+
+function downloadDocument(url: string) {
+  /* eslint-disable functional/immutable-data */
+  const link = document.createElement('a');
+  link.href = url;
+  link.target = '_blank';
+  link.rel = 'noreferrer';
+  link.click();
+  /* eslint-enable functional/immutable-data */
+};
+
+export function useDownloadDocument() {
```

Comment:
Maybe this can become a shared hook

Reply:
Maybe we can pass the legalFactDocumentDetails.url and the clearLegalFactDocumentData as props and macke the hook generic.
About the way we manage the downloads, do you have some advice?

Reply:
In fact no, otherwise I would have written it down. OK I do as you suggest ... even when I strongly disagree.

Reply:
We use the download also in NotificationDetail and it seems to be the same of your hook.
Why you disagree with the proposal to make it generic?

Reply:
Since I see the hook as too little and too dependent of the particular case (we should pass two props for a 6-line hook), is just a sensation. Maybe we could define some standard way to specify the info about a document to download, so that at least it would suffice with one prop. But as I told before, my ideas are not clear. And I tend to generalize only when I have a clear mindset. 
But I will proceed.

Reply:
I moved `useDownloadDocument` into pn-commons, have to say it looks better than I thought.